.github/workflows/check-file-updates.yaml

```diff
@@ -0,0 +1,43 @@
+name: Check config and readme updates
+on:
+  pull_request:
+    paths:
+      - "apis/**"
+jobs:
+  file-updates:
+    permissions:
+      pull-requests: write
+    name: Ensure generated files are included
```

Comment:
this is for "generated files and api-docs" both

Reply:
so do you want me to add generated files and docs are included?

Reply:
Yeah I did, both are generated. I can go both ways, though keeping it more generic does not make it less obvious to understand in terms of the what the goal of the job is.